EasyPost/Tracker.cs

```diff
@@ -102,31 +103,30 @@ public static TrackerCollection All(Dictionary<string, object> parameters = null
         /// </summary>
         /// <param name="id">String representing a Tracker. Starts with "trk_".</param>
         /// <returns>EasyPost.Tracker instance.</returns>
-        public static Tracker Retrieve(string id)
+        public static async Task<Tracker> Retrieve(string id)
         {
             Request request = new Request("trackers/{id}");
             request.AddUrlSegment("id", id);
 
-            return request.Execute<Tracker>();
+            return await request.Execute<Tracker>();
         }
 
         /// <summary>
         ///     Create a list of trackers
         /// </summary>
         /// <param name="parameters">A dictionary of tracking codes and carriers</param>
         /// <returns>True</returns>
-        public static bool CreateList(Dictionary<string, object> parameters)
+        public static async Task<bool> CreateList(Dictionary<string, object> parameters)
         {
-            Request request = new Request("trackers/create_list", RestSharp.Method.Post);
+            Request request = new Request("trackers/create_list", Method.Post);
             request.AddBody(new Dictionary<string, object>
             {
                 {
                     "trackers", parameters
                 }
             });
-            request.Execute<Tracker>();
+            return await request.Execute();
             // This endpoint does not return a response so we return true here
-            return true;
```

Comment:
Why are we switching this?

Reply:
Instead of always getting `true` back from the function, we now actually get a boolean based on whether the API call was successful or not.